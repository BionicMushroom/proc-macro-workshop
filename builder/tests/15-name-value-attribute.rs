use derive_builder::Builder;

#[derive(Builder)]
struct Test {
    #[builder = "whatever"]
    field: u32,
}

fn main() {}
