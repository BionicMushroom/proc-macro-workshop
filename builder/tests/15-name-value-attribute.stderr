error: expected `builder(each = "...")`
 --> tests/15-name-value-attribute.rs:5:7
  |
5 |     #[builder = "whatever"]
  |       ^^^^^^^^^^^^^^^^^^^^
